### ISSUE TITLE
Links in bubble titles not working in IE while scrolling

### ISSUE THREAD

**Issue by GoogleCodeExporter** [berusch/simile-widgets-old#374]
```
What steps will reproduce the problem?
1.The bubble title links does not work if you have to scroll to it in IE 
6; it works in Firefox
2.If scrolling is not required, the link works
3.

What is the expected output? What do you see instead?
The bubble link should work under all circumstances; but it does not when 
we scroll to it.


What version of the product are you using? On what browser and what
operating system?

IE 6 ; Windows 2003


Please provide any additional information below.

```

Original issue reported on code.google.com by `prana...@gmail.com` on 6 Jun 2010 at 5:45

**Comment by GoogleCodeExporter** [comment]
Original comment by `ryan...@csail.mit.edu` on 23 Jun 2011 at 10:29
- Added labels: **Project-Ajax**, **Project-Timeline**

**Comment by GoogleCodeExporter** [comment]
Original comment by `ryan...@csail.mit.edu` on 23 Jun 2011 at 10:34
- Removed labels: **Project-Ajax**

**Comment by GoogleCodeExporter** [comment]
```
I found the same problem on IE8. We have fixed the method 
getEventPageCoordinates inside simile-ajax-bundle.js as follows

SimileAjax.DOM.getEventPageCoordinates = function (A) {
    if (SimileAjax.Platform.browser.isIE) {

        var scrOfY = 0;
        var scrOfX = 0;

        if (document.body && (document.body.scrollLeft || document.body.scrollTop)) {
            //DOM compliant
            scrOfY = document.body.scrollTop;
            scrOfX = document.body.scrollLeft;
        } else if (document.documentElement && (document.documentElement.scrollLeft || document.documentElement.scrollTop)) {
            //IE6 standards compliant mode
            scrOfY = document.documentElement.scrollTop;
            scrOfX = document.documentElement.scrollLeft;
        }

        return { x: A.clientX + scrOfX, y: A.clientY + scrOfY };
    } else {
        return { x: A.pageX, y: A.pageY };
    }
};

it may also resolve the behavior described by Issue 248

Fixed By Christian Garofalo and Dmitrij Ivanov
```

Original comment by `dmitrij....@gmail.com` on 16 Sep 2011 at 11:41

Attachments:
- [simile-ajax-bundle.js](https://storage.googleapis.com/google-code-attachments/simile-widgets/issue-374/comment-3/simile-ajax-bundle.js)
